lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -1685,4 +1634,55 @@ bool MayaHydraSceneIndex::passNormalsToHydra()
     return val;
 }
 
+VtValue MayaHydraSceneIndex::_CreateDefaultMaterialFallback()
+{
+    static const MColor kDefaultGrayColor = MColor(0.5f, 0.5f, 0.5f) * 0.8f;
```

Comment:
Where does 0.8f come from?

Reply:
I don't know, it could be anything else, I guess I've picked it up from a maya shader, but that is arbitrary as this fallback shader should not be used.